test/bdd/steps/network.js

```diff
@@ -23,6 +24,42 @@ const bootstrapIdentity = {
     },
 };
 
+/**
+ * Unpacks cucumber dictionary into simple dictionary
+ * @param rawTable
+ */
+function unpackRawTable(rawTable) {
```

Comment:
Maybe in some future iterations we could move this function to /lib/utilities?

Reply:
Maybe, if there is a need for that.